.github/workflows/review-summary.yml

```
@@ -0,0 +1,142 @@
+          gh api \
+            -H "Accept: application/vnd.github+json" \
+            "/repos/${{ github.repository }}/pulls/${PR_NUMBER}/reviews/${REVIEW_ID}/comments" \
+            > review_comments.json
+
+          # Count comments
+          COMMENT_COUNT=$(jq length review_comments.json)
+          echo "comment_count=$COMMENT_COUNT" >> $GITHUB_OUTPUT
+
+      - name: Generate Summary
+        id: summary
+        run: |
```

actionlint (1.7.10)

"github.event.review.body" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)